webpack.config.js

```diff
@@ -1,56 +1,53 @@
 const HtmlWebpackPlugin = require('html-webpack-plugin');
-const PORT = process.env.PORT || 3000;
+const { resolve } = require('path');
+const webpack = require('webpack');
 
 module.exports = {
-  devtool: 'cheap-module-eval-source-map',
   entry: [
     'react-hot-loader/patch',
-    `webpack-dev-server/client?http://localhost:${PORT}`,
+    'webpack-dev-server/client?http://localhost:8080',
     'webpack/hot/only-dev-server',
-    './src/index.js',
+    './index.js',
   ],
+  output: {
+    filename: 'bundle.js',
+    path: resolve(`${__dirname}/dist`),
+    publicPath: '/',
+  },
+  context: resolve(__dirname, 'src'),
+  devtool: 'cheap-module-eval-source-map',
+  devServer: {
+    hot: true,
+    contentBase: resolve(__dirname, 'dist'),
+    publicPath: '/'
+  },
   module: {
     loaders: [
       {
-        test: /\.js(x?)$/,
+        test: /\.js$/,
```

Comment:
This is against airbnb react coding standard.

> Extensions: Use .jsx extension for React components.

https://github.com/airbnb/javascript/tree/master/react#naming